apps/passport-server/migrations/37_semphore_id.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE telegram_bot_conversations
+ADD column semaphore_id VARCHAR NOT NULL
```

Comment:
can we make this column be a foreign key to the `commitment` column of the `users` table?

Reply:
Done